src/packageurl/contrib/url2purl.py

```diff
@@ -73,45 +73,60 @@ def purl_from_pattern(type_, pattern, uri):
 
     if match:
         purl_data = {
-            field: value for field, value in match.groupdict().items()
+            field: value
+            for field, value in match.groupdict().items()
```

Comment:
Why is this on separate lines? it looks weird.

Reply:
It happened due to code formatting I will change it back